lib/PuppeteerSharp.Tests/NavigationTests/FrameGoToTests.cs

```diff
@@ -41,7 +40,14 @@ public async Task ShouldRejectWhenFrameDetaches()
             await waitForRequestTask;
             await Page.QuerySelectorAsync("iframe").EvaluateFunctionAsync("frame => frame.remove()");
             var exception = await Assert.ThrowsAsync<NavigationException>(async () => await navigationTask);
-            Assert.Equal("Navigating frame was detached", exception.Message);
+            Assert.Contains(
+                "Navigating frame was detached",
+                new[]
+                {
+                    "Navigating frame was detached",
+                    "Error: NS_BINDING_ABORTED",
+                    "net::ERR_ABORTED",
+                });
```

Comment:
Is this change intentional? The assertion is now always true as both subject and expectation are fixed values.

Reply:
This matches the change [upstream](https://github.com/puppeteer/puppeteer/blob/d0c5b8e08905f3802705a1a90d7cc8fa04bc82db/test/src/navigation.spec.ts#L808) 🤷

Reply:
No :)
The upstream asserts that `error.message` contains at least one of the three elements.
This change asserts that either of the three elements is `"Navigating frame was detached"`.

Reply:
This is the best I can come up with xUnit
```cs
Assert.Contains(new[] { exception.Message }, message => new[]
{
    "Navigating frame was detached",
    "Error: NS_BINDING_ABORTED",
    "net::ERR_ABORTED"
}.Any(error => message.Contains(error)));
```

Reply:
With FluentAssertions it can be written as
```cs
exception.Message.Should().ContainAny(
    "Navigating frame was detached",
    "Error: NS_BINDING_ABORTED",
    "net::ERR_ABORTED"
);
```

Reply:
Oops you're so right 😅
The test passes right? 😅